dev/cognitiveservices/data-plane/Language/generate.json

```diff
@@ -0,0 +1,230 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Generation Service",
+    "description": "The language generation service API is a suite of natural language generation (NLG) skills built with best-in-class Microsoft machine learning algorithms. The API can be used to generate summaries, marketing emails, conversation responses, questions and answers. Further documentation can be found in <a href=\"https://docs.microsoft.com/azure/cognitive-services/language-service/overview\">https://docs.microsoft.com/azure/cognitive-services/language-service/overview</a>.",
+    "version": "2023-04-01"
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://cognitiveservices.azure.com/.default"
+      ]
+    },
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language",
```

Comment:
Use the same hostTemplate as all other swaggers in this directory or validation will fail. That might me copying the `/language` prefix into each path below e.g., `/language:generate`.

Reply:
It is already the same "{Endpoint}/language" across "analyzetext.json", "analyzeconversations.json", etc. What did I miss?

Reply:
I was recalling from memory. I thought they put that into each endpoint. My mistake.